systemtests/src/main/java/com/github/streamshub/systemtests/utils/KafkaUtils.java

```diff
@@ -0,0 +1,31 @@
+package com.github.streamshub.systemtests.utils;
+
+import com.github.streamshub.systemtests.logs.LogWrapper;
+import io.skodjob.testframe.resources.KubeResourceManager;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePool;
+import io.strimzi.api.kafka.model.nodepool.KafkaNodePoolBuilder;
+import org.apache.logging.log4j.Logger;
+
+import static com.github.streamshub.systemtests.utils.WaitUtils.waitForKafkaReady;
+
+public class KafkaUtils {
+    private static final Logger LOGGER = LogWrapper.getLogger(KafkaUtils.class);
+    private KafkaUtils() {}
+
+    public static void scaleBrokers(String namespace, String kafkaName, int newBrokerCount, boolean wait) {
```

Comment:
Isn't it better to have the exact call in the method instead of encapsulating every step in some method in utils, which can be forgotten in next x months?
How many times you will actually scale the brokers? What if you want to scale controllers? You will create another method just for that?

Reply:
I would like to utilize scaling brokers for display purposes, data checking, reloading etc... i think it might be on par with extracting the methods.

Reply:
It is not that long, might go with raw methods instead of wrapping